web/twa-vis-framework/library/src/ts/ui/panel_handler.ts

```diff
@@ -119,6 +119,7 @@ class PanelHandler {
 		}
         document.getElementById("returnContainer").style.display = "none";
         window.currentFeature = null;
+        window.currentTimeIndex='1';
```

Comment:
You might have done this already but I was thinking this could be changed to being a map so that we can handle multiple variables/dimensions in the future.